src/output/outputMod.F90

```diff
@@ -1782,17 +1832,119 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        !================================================================
 
     case (cgnsCp)
-
+       ! mham:
+       ! [two] l 68, constants.F90
+       ! [gammaInf] l. 88 initializeFlow.F90
+       ! nonDimensional constant?
+       !   real(kind=realType), parameter :: two   = 2.0_realType
+       ! [MachCoef] l. 520 inputParam.F90
+       ! Mach number used to compute coefficients;
+       ! only relevant for translating geometries.
+       !
        ! Factor multiplying p-pInf
-
-       fact = two/(gammaInf*pInf*MachCoef*MachCoef)
+       fact = two/(gammaInf*pInf*MachCoef*MachCoef) ! = uInf2*rhoInf 
```

Comment:
you can also remove all the additions between 1845 - 1862 (revert the fact computation to the old version and remove comments basically)